server/runtime/src/main/java/org/infinispan/server/configuration/DataSourceConfigurationBuilder.java

```diff
@@ -88,6 +88,11 @@ public DataSourceConfigurationBuilder backgroundValidation(long backgroundValida
       return this;
    }
 
+   public DataSourceConfigurationBuilder idleValidation(long idleValidation) {
```

Comment:
Since you are using `validateOnAcquisition` everywhere :) 
```suggestion
   public DataSourceConfigurationBuilder validateOnAcquisition(long idleValidation) {
```

Reply:
or change everywhere to `idleValidation`.

Reply:
Fixed